include/repository.php

```diff
@@ -794,6 +794,11 @@ function post_github_pull_request_comment($projectid, $pull_request, $comment, $
 /** Find changes for a "version only" update. */
 function perform_version_only_diff($update, $projectid)
 {
+    // Return early if we don't have a current revision.
+    if (empty($update->Revision)) {
```

Comment:
If the previous build did not have a revision, you would need to exit early too. I am seeing that error on my instance now.